src/transform-sdk/js/js_vm.cc

```diff
@@ -394,6 +395,50 @@ class console {
     FILE* _info_stream;
     FILE* _err_stream;
 };
+
+class process {
+    using map_t = std::unordered_map<std::string, std::string>;
```

Comment:
Can we add `NODE_ENV=production` if there is no user environment variable set for the `NODE_ENV` key?

Reply:
sure